sos/report/plugins/redis.py

```diff
@@ -9,23 +9,21 @@
 #
 # See the LICENSE file in the source distribution for further information.
 
-from sos.report.plugins import Plugin, RedHatPlugin
+from sos.report.plugins import Plugin, SCLPlugin
 
 
-class Redis(Plugin, RedHatPlugin):
-
-    short_desc = 'Redis, in-memory data structure store'
+class Redis(Plugin, SCLPlugin):
+    """Redis, in-memory data structure store
+    """
```

Comment:
Small regression, the `short_desc` attribute is now used instead of docstrings. Let's add that back.

Reply:
New commit ad3cb2b15dfa6d195c314b6bd44a1825006b9284 proposed, is it ok for you?